### DISCUSSION TITLE
useLoader deprecated? What is the recommended way to get data from load

### DISCUSSION THREAD

**Discussion by erickarnis-tb** [TanStack/router#812] [Q&A]
useLoader is still in the docs and examples but trying to  import it from `@tanstack/react-router` gives a type error saying that it's not exported.  ` component: ({ useLoader }) => ...` similarly gives a type error saying useLoader  does not exist on type RouteProps. I am using Apollo Client but I'm sure this question is broadly applicable.  I am using version 0.0.1-beta.211.

Say I have this route:

```javascript
const organizationsRoute = new Route({
  component: Organizations,
  getParentRoute: () => authRoute,
  load: async () => await client.query({ query: ALL_ORGANIZATIONS }),
  path: "/organizations",
})
```

and this component

```javascript
export const Organizations = () => {
	return <p>Example</p>
}
```

How should I get the results of the query in load into the component?

**Comment by tannerlinsley** [comment]
The documentation is out of date as of about a week ago. The internals of the library have moved to a 100% react architecture, and the caching feature has been removed from the library in favor of external cashes like Query. I’m almost done re-factoring the internals, then I will update all of the documentation. Stay tuned.

**Reply by erickarnis-tb** [reply]
Sounds great! Is there any way you or someone else could add a version number to the docs so people know which version they apply to? I was struggling with it and I thought I was doing something wrong.
